lms/views/api/lti.py

```diff
@@ -104,7 +104,8 @@ def __call__(self, score=None, request_body=None):
         # an existing submission then the existing submission is updated
         # rather than creating a new submission.
         request_body["submissionDetails"] = {
-            "submittedAt": datetime.datetime(2001, 1, 1, tzinfo=timezone.utc)
+            "submittedAt": self.request.parsed_params["submitted_at"]
+            or datetime.datetime(2001, 1, 1, tzinfo=timezone.utc)
```

Comment:
Keeping the default for the same reason it's optional. Can be made mandatory and remove this `or` in a second deploy.